lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1568,7 +1572,7 @@ static bool ParseClangImporterArgs(ClangImporterOptions &Opts,
 
   // Forward the FrontendOptions to clang importer option so it can be
   // accessed when creating clang module compilation invocation.
-  if (FrontendOpts.EnableCaching)
+  if (FrontendOpts.EnableCaching || FrontendOpts.UseCASBackend)
```

Comment:
Nit: I will create a method for this, like `requiresCAS()` to keep all the logic in sync (like with `setupCASIfNeeded`)